### ISSUE TITLE
[Spike] Lighthouse solution (in progress of editing)

### ISSUE THREAD

**Issue by isabelolson** [PublicDataWorks/complaint-manager#218]
### Business Context:
We want a way of automatically doing accessibility reviews/analysis of our app. We cannot run lighthouse against Demo in CircleCI anymore since heroku no longer supports free hosting. This spike serves as an investigation for how we should run the accessibility reviews of our app. 

Possible Solutions to be investigated:
- Finding an alternative to heroku that is free (for hosting demo)
- Run lighthouse against CI instead of Demo (found an old card similar to this solution? [LINK](https://github.com/PublicDataWorks/complaint-manager/issues/121)) Think there was an issue with being able to do this since we couldn't login from CircleCI??
- Just have a reoccurring card every month for running lighthouse against CI
- Alternatives to lighthouse?  Axe DevTools Chrome Extension

Questions to be Answered:

Timebox:
[decide during grooming]

**As a**
**I want**
**So that** 

### In Scope
1.
2.
3.

### Out of Scope
1.
2.
3.

### Open Questions
1.
2.
3.

## Acceptance Criteria

1. Scenario: 
Given
When
Then

2. Scenario: 
Given
When
Then

### Checklist (should be reviewed before grooming as the BA or Pair are writing the card—NOT reviewed for the first time at grooming)
0. Discuss the technical implementation of this card.
1. Discuss how this feature would be included in or affect the Audit Log.
2. In respect to security, what could go wrong? Brainstorm threats using STRIDE model, especially if this card introduces a new technical architecture or design pattern. (TIMEBOX: 7 mins) https://docs.google.com/document/d/18AFl1d3SRieI4d_8R3GyEazqQOycLNDHGN_vOYIHuiE/edit#heading=h.aio9zb5j63wl
3. Ensure any open questions have been resolved with stakeholders.
4. Determine if documentation should be referenced or created.
5. Ensure to include all necessary mockups to this card.
6. Discuss what success/failure user feedback messages would be present, if any occur (front end/back end).
7. Discuss why this feature would not be implemented behind a toggle / what would happen if this card was partially done while a release branch was being cut.
8. If this card is part of a sequence of cards, discuss if this card should be blocked until the testing of previous  cards in the sequence have been finished.
9. Have we added edge cases as ACs that we're considered from a QA perspective
10. Does this card overlap with any entities outlined in [API Housekeeping](https://www.pivotaltracker.com/story/show/173963545)?  

**Notes:**

**Comment by rnair-tw** [comment]
With new design, accessibility should be included as part of process. Do discuss if tests should be included in pipeline or as part of design process.
